UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/SceneImporter/ImporterMaterials.cs

```diff
@@ -752,6 +753,27 @@ protected virtual Task ConstructMaterialImageBuffers(GLTFMaterial def)
 				}
 			}
 
+
+			if (def.Extensions != null && def.Extensions.ContainsKey(KHR_materials_clearcoat_Factory.EXTENSION_NAME))
+			{
+				var clearCoatDef = (KHR_materials_clearcoat) def.Extensions[KHR_materials_clearcoat_Factory.EXTENSION_NAME];
+				if (clearCoatDef.clearcoatTexture != null)
+				{
+					var textureId = clearCoatDef.clearcoatTexture.Index;
+					tasks.Add(ConstructImageBuffer(textureId.Value, textureId.Id));
```

Comment:
Have you tested clearcoat export roundtrip as well? (even when the shader currently doesn't support it the data should roundtrip correctly and still be there on export /reimport)